### PR TITLE
Stats: change /calypso/images -> calypso/assets/images to allow correct resolving of assets

### PR DESCRIPTION
**Pull request by kangzj**
#### Proposed Changes
The PR changes all assets referred by the Stats App from `/calypso/images/...` to `calypso/assets/images/...` which allows correct resolving by the loaders and also has benefits for cache busting. Assets in `/static` are retained because we are not sure where they are also referenced.

Also a note for the team: in the future, please do not add more assets to `/static`, the correct location is `client/assets/`.

More information: https://github.com/Automattic/wp-calypso/blob/954e16bee68d10e4031f8d8fb22b3546abf5c687/static/images/README.md

#### Testing Instructions
* Open a site with Jetpack plugin
* Open `/wp-admin/admin.php?page=jetpack#/traffic?term=jetpack%20stats`
* Expand the Jetpack Stats section
* Open browser console and run `document.querySelector('#jp-settings-site-stats').querySelectorAll('.jp-form-fieldset')[1].setAttribute('style','color:red')`
* Turn on the new Stats experience (color red toggle)
* Open `/wp-admin/admin.php?page=stats`
* Ensure the page looks reasonable, no assets 404

* Checkout the branch run `yarn start`
* Open `http://calypso.localhost:3000/` for Stats pages
* Ensure no assets 404

image

image

#### Pre-merge Checklist

- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

Related to #

### CODE REVIEW

File: client/my-sites/customer-home/cards/features/stats/index.jsx [hunk 1]
```diff
@@ -105,7 +105,7 @@ export const StatsV2 = ( {
 				{ ! isSiteUnlaunched && ! isLoading && views === 0 && (
 					<div className="stats__empty">
 						<div className="stats__empty-illustration">
-							<img src="/calypso/images/stats/illustration-stats-intro.svg" alt="" />
+							<img src="calypso/assets/images/stats/illustration-stats-intro.svg" alt="" />
```
Comment (dognose24) [inline client/my-sites/customer-home/cards/features/stats/index.jsx:108]: Could this src be imported on the top of the file?

File: client/my-sites/stats/stats-post-detail/index.jsx [hunk 2]
```diff
@@ -162,7 +162,7 @@ class StatsPostDetail extends Component {
 						action={ translate( 'Get more traffic!' ) }
 						actionURL="https://wordpress.com/support/getting-more-views-and-traffic/"
 						actionTarget="blank"
-						illustration="/calypso/images/stats/illustration-stats.svg"
+						illustration="calypso/assets/images/stats/illustration-stats.svg"
```
Comment (dognose24) [inline client/my-sites/stats/stats-post-detail/index.jsx:165]: Could this svg be imported on the top of the file?

**Review by dognose24 — APPROVED** [review]
LGTM! I left a few comments, which will not be blockers.

File: client/my-sites/customer-home/cards/features/stats/index.jsx (hunk 1 above)
Comment (kangzj) [reply client/my-sites/customer-home/cards/features/stats/index.jsx:108]: The reason the assets are imported in other places is that, if we don't do it, the asset wouldn't be resolved correctly. For react the assets are resolved automatically, and I don't find it necessary to import manually - they are going to be used only once anyway. What do you think?

Comment (dognose24) [reply client/my-sites/customer-home/cards/features/stats/index.jsx:108]: Thanks for the explanation, @kangzj! Sounds reasonable to me! 😄

Comment (a8ck3n) [reply client/my-sites/customer-home/cards/features/stats/index.jsx:108]: This change breaks the image in my testing. I'm seeing a 404 in the console for it. Reverting to the previous URL allows the image to load.

MissingSVG

You can reproduce by visiting the My Home for a site without any recent traffic. That card is 3rd from the top on my test site.

cc: @kangzj

File: client/my-sites/stats/stats-post-detail/index.jsx (hunk 2 above)
Comment (a8ck3n) [reply client/my-sites/stats/stats-post-detail/index.jsx:165]: Likewise with this change. Visiting the My Home page triggers a 404 for this SVG asset.

**Comment by kangzj** [comment]
Hi @a8ck3n This is some good catch. For some reason, `src` is not loaded properly by webpack (usually loaded by `html-loader`). Anyway I'll spin a fix. Thanks again for catching it 👍

**Pull request merged**